app.config.js

```diff
@@ -1,7 +1,8 @@
 const configs = [
   require('cozy-scripts/config/webpack.bundle.default'),
   require('cozy-scripts/config/webpack.config.css-modules'),
-  require('./config/webpack.config.piwik')
+  require('./config/webpack.config.piwik'),
+  require('./config/webpack.config.cozy-ui')
 ]
 
```

Comment:
Why not using an alias here instead of adding `/transpiled` everywhere?

Reply:
If an alias is working, that's also fine with me a easy to test/remove later